sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/EventGridPublisherClientBuilder.java

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
```

Comment:
License headers should be the following in all Java files:
https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html#java
```java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```